bp-web-client/angular-app/src/app/components/app-contact/app-contact.component.ts

```diff
@@ -27,6 +27,7 @@ import { Router } from "@angular/router";
 export class ContactComponent
   implements IDeactivateComponent, OnInit, OnDestroy
 {
+  isError: boolean = false;
```

Comment:
Add the appropriate variable scope here.  I.E (public, private, and or protected)

Reply:
Added Public Scoping